.github/workflows/integration-test.yaml

```diff
@@ -42,16 +42,6 @@ jobs:
     runs-on: ${{ inputs.runs-on }}
     steps:
       - uses: actions/checkout@v3
-      - name: Setup powershell
```

Comment:
please don't remove this, the setup python action requires powershell, also there's other non powershell stuff this is installing. It should be renamed to `Setup self hosted`

Reply:
Restored in commit 445b6432c8e9b997ed1e953cb0b5634ad775e65b, and renamed in commit d6042ea8474cf40d1994c9ec68df69d8b89203e9.